eevee/metrics/classification.py

```diff
@@ -17,6 +18,12 @@ def intent_report(
     # for cases where we are seeing NaN values popping up.
     df[['intent_x', 'intent_y']] = df[['intent_x', 'intent_y']].fillna(value="_")
 
+    # aliasing intents
+    if intent_aliases is not None:
+        alias_dict = {intent: alias for alias, intent_list in intent_aliases.items() for intent in intent_list}
```

Comment:
`intent_aliases` - which is built from reading the `alias.yaml` - stores data as a `List[Dict[str, List]]`. this reverses the key and values, and the data to a `Dict` for ease of use